repl/src/test/scala/org/apache/spark/repl/ExecutorClassLoaderSuite.scala

```diff
@@ -231,6 +231,7 @@ class ExecutorClassLoaderSuite
       .setMaster("local")
       .setAppName("executor-class-loader-test")
       .set("spark.network.timeout", "11s")
+      .set("spark.network.timeoutInterval", "11s")
```

Comment:
`spark.network.timeoutInterval` default value is 60s, `spark.network.timeoutInterval` should <= `spark.network.timeout`

Reply:
without this line , this suite failed cause by :
```
requirement failed: spark.network.timeoutInterval should be less than or equal to spark.storage.blockManagerHeartbeatTimeoutMs.
java.lang.IllegalArgumentException: requirement failed: spark.network.timeoutInterval should be less than or equal to spark.storage.blockManagerHeartbeatTimeoutMs.

```